ydb/core/kqp/runtime/kqp_compute.cpp

```diff
@@ -102,6 +115,14 @@ class TKqpIndexLookupJoinWrapper : public TMutableComputationNode<TKqpIndexLooku
             Both
         };
 
+        TState& GetState() const {
+            auto& result = Ctx.MutableValues[StateIndex];
+            if (!result.HasValue()) {
```

Comment:
As far as I know there should be `result.IsInvalid()` instead of `result.HasValue()`
https://github.com/ydb-platform/ydb/pull/8555